### PR TITLE
isaac-server: add release 1.5.1 and 1.5.2

### PR DESCRIPTION
**Pull request by psychocoderHPC**
Add latest releases of isaac-server and adjust jansson requirements.

### CODE REVIEW

File: var/spack/repos/builtin/packages/isaac-server/package.py [hunk 1]
```diff
@@ -29,7 +31,8 @@ class IsaacServer(CMakePackage):
 
     depends_on('cmake@3.3:', type='build')
     depends_on('jpeg', type='link')
-    depends_on('jansson@:2.9', type='link')
+    depends_on('jansson@:2.9', type='link', when='@:1.5.1')
+    depends_on('jansson', type='link', when='@1.5.2:')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/isaac-server/package.py]: This can be simplified as:
```suggestion
    depends_on('jansson', type='link')
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Only a minor comment, otherwise LGTM

File: var/spack/repos/builtin/packages/isaac-server/package.py (hunk 1 above)
Comment (psychocoderHPC) [reply var/spack/repos/builtin/packages/isaac-server/package.py]: Thanks, I was not sure if this is equivalent, I changed it to the suggested version.

Comment (alalazo) [reply var/spack/repos/builtin/packages/isaac-server/package.py]: No worries, this is equivalent. The directive states that `isaac-server` has an unconditional dependency on `jansson`. The directive above adds more constraints if the version of `isaac-server` is 1.5.1 or less.

**Review by alalazo — APPROVED** [review]

**Pull request merged**